org.freedesktop.Sdk.Extension.texlive.yml

```diff
@@ -102,8 +126,14 @@ modules:
       - --enable-bundled-libs
     sources:
       - type: archive
-        url: https://github.com/mgieseki/dvisvgm/archive/2.11.1.tar.gz
-        sha256: 71b772d97abe04bc0b9a4dbd0077f4224ade7ea0aac325f061e4c0037d31020c
+        url: https://github.com/mgieseki/dvisvgm/releases/download/2.12/dvisvgm-2.12.tar.gz
+        sha256: 90cb6e117286f98f03e2f8e3f5526cf1d370686a7186e57a031a506bf6ead0f4
+        x-checker-data:
+          type: json
```

Comment:
Why do you not want to use `anitya` for dvisvgm [https://release-monitoring.org/project/130020/](https://release-monitoring.org/project/130020/)? Is there a particular reason?

Reply:
I noticed that some maintainers that are affiliated with Flathub prefer to check GitHub directly, so I'm following their example.  
I actually prefer to avoid GitHub if I can due to the REST API hourly limit for anonymous requests.